ci/goreleaser.yml

```diff
@@ -0,0 +1,36 @@
+name: goreleaser^
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
```

Comment:
Should this include pushing to version tags, or is that not allowed? 🤔

Reply:
It the PR checklist tis written it should be like this:

```
Should run on push to branches: [ $default-branch ] and pull_request to branches: [ $default-branch ].
```

The other actions are all the same.

Reply:
I think it should be only the tags, as its just the goreleaser workflow... otherwise it'll try to release PRs and other branches as well...

Reply:
is 

```
    tags:
      - '*'
```
okay, @caarlos0?

Reply:
yeah, that should be ok

Reply:
done